cluster/saltbase/salt/kubelet/default

```diff
@@ -3,23 +3,10 @@
   {% set daemon_args = "" -%}
 {% endif -%}
```

Comment:
here be dragons... need an ack from someone familiar with the salt setup on the kubelet (and the implications of changing these for upgrade/downgrade to previous point release)